### PR TITLE
Transporter upload tests + bug fixes

### PR DESCRIPTION
**Pull request by ChGen**
Transporter http3 upload tests + bug fixes. Also updated docker configuration and fixed running tests in CI.

### CODE REVIEW

**Comment by ChGen** [comment]
@dgarus , please review.

File: mvn-resolver-transport-http3/src/main/java/com/artipie/aether/transport/http3/HttpTransporter.java [hunk 1]
```diff
@@ -199,7 +203,18 @@ private ContentResponse makeRequest(HttpMethod method, TransportTask task, Reque
         final Request request = this.client.newRequest(url);
         final ContentResponse response;
         try {
-            response = request.method(method).body(bodyContent).send();
+            response = request.method(method).headers(httpFields -> {
+                System.err.printf("\tCustom HEADER HttpTransporter.makeRequest() called! fields: %d; URL: %s%n", httpFields.size(), url);
```
Comment (olenagerasimova) [inline mvn-resolver-transport-http3/src/main/java/com/artipie/aether/transport/http3/HttpTransporter.java:207]: maybe remove this debug message?

**Review by olenagerasimova — APPROVED** [review]
just one suggestion, thanks

**Comment by ChGen** [comment]
@olenagerasimova , thanks, please review.

**Review by olenagerasimova — APPROVED** [review]

File: mvn-resolver-transport-http3/src/test/java/com/artipie/aether/transport/http3/MavenResolverIT.java [hunk 2]
```diff
@@ -111,39 +118,87 @@ public void testJettyLocalhostConnection() throws Exception {
...
+    }
+
+    @Test
+    public void testTransporterPutAnon() throws Exception {
+        final String repo = "https://localhost:7445/";
+        restartPutServer();
+        final HttpTransporterFactory factory = new HttpTransporterFactory();
+        final PutTask task = new PutTask(URI.create("test1")).setListener(new TransportListener() {});
+        final String testData = "test data";
+        task.setDataString(testData);
```
Comment (dgarus) [inline mvn-resolver-transport-http3/src/test/java/com/artipie/aether/transport/http3/MavenResolverIT.java:149]: Maybe would it be better to use binary test data? 
WDYT?

**Review by dgarus — CHANGES_REQUESTED** [review]
thanks! there is one question

File: mvn-resolver-transport-http3/src/test/java/com/artipie/aether/transport/http3/MavenResolverIT.java (hunk 2 above)
Comment (ChGen) [reply mvn-resolver-transport-http3/src/test/java/com/artipie/aether/transport/http3/MavenResolverIT.java:149]: Good point, I'll switch to binary test data.

**Review by dgarus — APPROVED** [review]

**Pull request merged**